### ISSUE TITLE
Storybook - YearChart error

### ISSUE THREAD

**Issue by sebastiangelotte** [PaulieScanlon/gatsby-theme-gatstats#31]
I love the theme - nice work!

I found an error in the Storybook YearChart section:
![image](https://user-images.githubusercontent.com/22306228/72414660-0c93c900-3773-11ea-94dc-25417bd0e00a.png)

Error:
```
TypeError: Cannot read property 'map' of undefined
    at YearChartComponent (https://gatsby-theme-gatstats.netlify.com/storybook/main.4a424575b496f6b1c88b.bundle.js:1:45739)
...
```

**Comment by PaulieScanlon** [comment]
Oh blast! Thanks @sebastiangelotte i made some changes to that component recently, and must have forgotten to spin up Storybook, it’s probably a mock prop issue. Leave it with me.

**Comment by PaulieScanlon** [comment]
@sebastiangelotte hey! fixed in release `0.0.54`
